features/support/env.rb

```diff
@@ -150,5 +150,25 @@ def extract_process_id(data)
   end
 
   expect(binaries.size).to eq(1)
-  expect(File.executable?(binaries[0])).to eq(true)
+  expect(File.executable?(binaries.first)).to eq(true)
+end
+
+Then(/^the binary "([^"]*)" should be installed with version "([^"]*)"$/) do |binary, version|
+  binary_path = "#{ENV['VAGRANT_HOME']}/data/service-manager/bin/#{version}/#{binary}"
```

Comment:
I was wrong, we cannot build the path directly. This test fails now, there is one more level after 'bin' which depicts the service. We either would need to pass the service as well to build the full path or we indeed have to do some path matching. Maybe it is more explicit to so something like with something like:
"Then(/^the binary "([^"]*)" of service "([^"]*)" should be installed with version "([^"]*)"$/)"? I just want to make sure we don't get any false positives by using a recursive scan approach.